lib/api/cli/index.js

```diff
@@ -1,91 +1,109 @@
+/*
+ * Copyright 2017 Kaliop SAS
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
 'use strict';
 
-let
+const
   Action = require('./action'),
   InternalBroker = require('../../services/internalBroker'),
   Request = require('kuzzle-common-objects').Request;
 
 /**
+ * @class CliActions
  * @param {Kuzzle} kuzzle
- * @constructor
  */
-function CliActions (kuzzle) {
-  this.actions = {
-    adminExists: new Action(),
-    clearCache: new Action(),
-    cleanDb: new Action(),
-    createFirstAdmin: new Action(),
-    data: new Action(),
-    dump: new Action(),
-    managePlugins: new Action({
-      timeout: 1000,
-      timeOutCB: function cliTimeOutCB () {
-        if (!this.maxTimeout) {
-          this.maxTimeout = 5 * 60 * 1000;
-          this.spent = this.timeout;
+class CliActions {
+  constructor(kuzzle) {
+    this.kuzzle = kuzzle;
+
+    this.actions = {
+      adminExists: new Action(),
+      clearCache: new Action(),
+      cleanDb: new Action(),
+      createFirstAdmin: new Action(),
+      data: new Action(),
+      dump: new Action(),
+      managePlugins: new Action({
+        timeout: 1000,
+        timeOutCB: () => {
+          if (!this.maxTimeout) {
+            this.maxTimeout = 5 * 60 * 1000;
+            this.spent = this.timeout;
+          }
+
+          this.spent += this.timeout;
+
+          if (this.spent < this.maxTimeout) {
+            process.stdout.write('.');
+            this.initTimeout();
+          }
+          else {
+            console.error(`ERROR: No response from Kuzzle within ̀${this.maxTimeout / 1000}s. Exiting`); // eslint-disable-line no-console
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Split this 138 characters long line (which is greater than 80 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ALineLength)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ATrailingComment)